pkg/remote/trans/default_server_handler.go

```diff
@@ -273,7 +275,11 @@ func (t *svrTransHandler) writeErrorReplyIfNeeded(
 		// conn is closed, no need reply
 		return
 	}
-	if methodInfo, _ := GetMethodInfo(ri, t.svcInfo); methodInfo != nil {
+	svcInfo := recvMsg.ServiceInfo()
+	if svcInfo == nil {
+		return
```

Comment:
cannot return directly, the logic below must be executed.

Reply:
@jayantxie If the logic below is executed when svcInfo is nil, there will be nil pointer reference panic... is it ok to pick a random svcInfo...? (the error message can be replaced though)

Reply:
Actually even if the svc info is nil, the logic below won't panic because its not used.

Reply:
@jayantxie I think It's used in GetMethodInfo()

Reply:
Actually not, exception data doesn't need svc info to initialize it.